src/main/java/com/gooddata/dataload/processes/ProcessService.java

```diff
@@ -481,6 +531,36 @@ private DataloadProcess postProcess(DataloadProcess process, File processData, U
         }
     }
 
+    private FutureResult<DataloadProcess> postProcess(DataloadProcess process, String appstorePath, URI postUri, HttpMethod method) {
+        try {
+            process.setPath(appstorePath);
+            ResponseEntity<String> exchange = restTemplate.exchange(postUri, method, new HttpEntity<>(process), String.class);
+            if (exchange.getStatusCode() == HttpStatus.ACCEPTED) { //deployment worker will create process
+                String uri = mapper.readTree(exchange.getBody()).findPath("poll").asText();
+                return new PollResult<>(this, new SimplePollHandler<DataloadProcess>(uri, DataloadProcess.class) {
+
+                    @Override
+                    public void handlePollException(GoodDataRestException e) {
+                        throw new GoodDataException("Creating process failed", e);
+                    }
+                });
+            } else if (exchange.getStatusCode() == HttpStatus.OK) { //object has been found in package registry, deployment worker is not triggered
+                final DataloadProcess dataloadProcess = mapper.readValue(exchange.getBody(), DataloadProcess.class);
```

Comment:
cannot use extractData, because it accepts as argument ClientHttpResponse and I cannot pass ResponseEntity to it